vault/resource_approle_auth_backend_role_secret_id.go

```diff
@@ -330,8 +345,8 @@ func approleAuthBackendRoleSecretIDExists(d *schema.ResourceData, meta interface
 	return resp != nil, nil
 }
 
-func approleAuthBackendRoleSecretIDID(backend, role, accessor string, wrapped bool) string {
-	if wrapped {
+func approleAuthBackendRoleSecretIDID(backend, role, accessor string, wrapped bool, reuse_wrapping_token bool) string {
```

Comment:
Go variable/argument name should be camel cased.
```suggestion
func approleAuthBackendRoleSecretIDID(backend, role, accessor string, wrapped bool, reuseWrappingToken bool) string {
```